src/main/java/com/inventory/structures/StackStruct.java

```diff
@@ -0,0 +1,67 @@
+package com.inventory.structures;
+
+import com.inventory.model.Category;
+import com.inventory.InventoryApplication;
+import com.inventory.repository.CategoryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+//import java.util.Scanner;
+import java.util.Stack;
+
+
+public class StackStruct extends Category {
+
+    public static void main(String[] args){
+
+        CategoryRepository categoryRepository = null;
+
+        //Scanner scanner = new Scanner(System.in);
+        Stack<Category> stack = new Stack<Category>();
+
+        //Populate the stack with user input
```

Comment:
instead of individual population, why don't you return the stack so you can add the response from the `CategoryImplementation` class

Reply:
okay